ydb/public/lib/ydb_cli/commands/ydb_service_scheme.cpp

```diff
@@ -543,6 +550,50 @@ int TCommandDescribe::DescribeReplication(const TDriver& driver) {
     return PrintDescription(this, OutputFormat, result, &TCommandDescribe::PrintReplicationResponsePretty);
 }
 
+int TCommandDescribe::PrintViewResponsePretty(const NKikimrSchemeOp::TViewDescription& desc) const {
+    Cout << Endl << "Query text: " << desc.GetQueryText() << Endl;
+    return EXIT_SUCCESS;
+}
+
+int TCommandDescribe::DescribeView(TConfig& config) {
+    const NKikimr::NGRpcProxy::TGrpcError* transportError = nullptr;
+    NKikimrClient::TResponse response;
+    {
+        auto clientConfig = NYdbGrpc::TGRpcClientConfig(config.Address);
+        if (config.EnableSsl) {
+            clientConfig.EnableSsl = config.EnableSsl;
+            clientConfig.SslCredentials.pem_root_certs = config.CaCerts;
+        }
+        NKikimr::NGRpcProxy::TGRpcClient client(clientConfig);
+
+        NKikimrClient::TSchemeDescribe request;
+        request.SetPath(Path);
+        if (config.SecurityToken) {
+            request.SetSecurityToken(config.SecurityToken);
+        }
+
+        client.SchemeDescribe(request,
+            [&transportError, &response](const NKikimr::NGRpcProxy::TGrpcError* te, const NKikimrClient::TResponse& r) {
+                if (te) {
+                    transportError = te;
+                } else {
+                    response = r;
+                }
+            }
+        );
+    }
+    if (transportError) {
+        throw yexception() << "grpc transport error message: " << transportError->first
+            << ", code: " << transportError->second;
+    }
+    if (response.status() != NKikimr::NMsgBusProxy::MSTATUS_OK) {
+        throw yexception() << "scheme describe error status: " << response.status()
+            << ", issues: " << NYql::IssuesFromMessageAsString(response.issues());
+    }
```

Comment:
I haven't written unit tests yet, because I'm not sure this approach would be the accepted one, but if you are interested in the output of the `ydb scheme describe` command, here is how it looks like:

```
...$ ydb scheme describe v
<view> v

Query text: select * from t

...$ ydb scheme describe vvv
Status: SCHEME_ERROR
Issues: 
<main>: Error: Path not found
```

I'm not sure where the error message is generated. You can notice that its format is different from what I have written for the `response.status() != MSTATUS_OK` case.

However, error handling is covered 🤷